e2e-polybft/helpers_test.go

```diff
@@ -77,3 +78,17 @@ func getRootchainValidators(relayer txrelayer.TxRelayer, checkpointManagerAddr,
 
 	return validators, nil
 }
+
+func hasValidatorSealed(srv *framework.TestServer, firstblock, lastblock uint64, validators polybft.AccountSet, validatorAddr ethgo.Address) bool {
+	for i := firstblock + 1; i <= lastblock; i++ {
+		block, _ := srv.JSONRPC().Eth().GetBlockByNumber(ethgo.BlockNumber(i), false)
```

Comment:
Could you add errcheck there?

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/8de4d6a725d75f5819e1071cfac7a5ffbab412fb